services/motion/builtin/move_request.go

```diff
@@ -94,7 +94,6 @@ func (mr *moveRequest) Plan(ctx context.Context) (state.PlanResponse, error) {
 	}
 	mr.planRequest.StartConfiguration = map[string][]referenceframe.Input{mr.kinematicBase.Kinematics().Name(): inputs}
 
-	// TODO(RSDK-5634): this should pass in mr.seedplan and the appropriate replanCostFactor once this bug is found and fixed.
```

Comment:
@raybjork @biotinker given the fact that we never (to my knowledge) tested that replanning using a seed plan works should I simply delete this comment & we can implement replanning with seed plans at some point in the future if it ends up being necessary?